mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -257,19 +275,13 @@ view: subscriptions__retention {
     ]
   }
 
-  dimension: retention_type {
-    sql:
-    IF(${retention_month} = 0, "first month", "subsequent months");;
-  }
-
-  dimension: period_end_7_day_window {
+  dimension: primary_key {
+    primary_key: yes
     hidden: yes
-    sql: GENERATE_DATE_ARRAY(${period_end_date}, ${period_end_date} + 6);;
+    sql: ${subscriptions.subscription_id} || ${age_in_months};;
   }
 
-  measure: retained {
-    description: "Subscriptions retained past the end of the 0-based retention_month"
-    type: number
-    sql: COUNTIF(${retention_month} < ${subscriptions.months_retained});;
+  measure: count {
+    type: count
```

Comment:
this needs a separate count from `subscriptions.count` because when averaging churn over each month you have to count subscriptions multiple times.